### DISCUSSION TITLE
Help getting started

### DISCUSSION THREAD

**Discussion by kimgandy** [community/community#22649] [New to GitHub]
<p>Hi I am new here. I am currently applying to a class for learning computer code. I literally know nothing about code but halfway through the application this is the message that I got</p>
<p>Explain a complex topic from your area of expertise in 500 words or fewer. This writing should be original to this application. Citations are welcome, but not necessary.<br>
Post the text as a private gist on <a href="http://gist.github.com" rel="nofollow noopener">http://gist.github.com</a> with the language set to Markdown. Make sure the extension of the file is set to .md<br>
Please ensure the document contains no formatting other than Markdown. Take care when copy/pasting from other text editors.<br>
Insert the link to your document in the text field above.</p>
<p>and now I am here trying to figure out how to fulfill the request. I have tried reading how to and watching tutorial videos but I am still puzzled can someone help me please?</p>

**Comment by airtower-luna** [comment]
<p>What exactly is it that you’re puzzled about? If you go to <a href="https://gist.github.com/">https://gist.github.com/</a> while logged in you should automatically get the form to create a new gist. Is anything there unclear? Or are you puzzled by Markdown?</p>

**Comment by kimgandy** [comment]
<p>thank you so much for taking the time to reply. Not sure where to start, do I need to know how to code already in order to fulfill the request or is this just something that i can just type into and then send them a link? Thanks for your help.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>No coding necessary at this stage. Would be very weird of them to require that for an introductory class. :wink:</p>
<p>So, let me break down that task for you:</p>
<aside class="quote no-group" data-username="kimgandy" data-post="1" data-topic="131604">
<div class="title">
<div class="quote-controls"></div>
 kimgandy:</div>
<blockquote>
<p>Explain a complex topic from your area of expertise in 500 words or fewer. This writing should be original to this application. Citations are welcome, but not necessary.</p>
</blockquote>
</aside>
<p>Basically: Write an essay about a topic you’re familiar with.</p>
<aside class="quote no-group" data-username="kimgandy" data-post="1" data-topic="131604">
<div class="title">
<div class="quote-controls"></div>
 kimgandy:</div>
<blockquote>
<p>Post the text as a private gist on <a href="http://gist.github.com">http://gist.github.com</a> with the language set to Markdown. Make sure the extension of the file is set to .md</p>
</blockquote>
</aside>
<p>What they’re really asking here: Can you read a form and follow instructions? :sweat_smile:</p>
<p>One thing about gists that may not be immediately obvious: You can edit them, and they keep a record of such changes.</p>
<aside class="quote no-group" data-username="kimgandy" data-post="1" data-topic="131604">
<div class="title">
<div class="quote-controls"></div>
 kimgandy:</div>
<blockquote>
<p>Please ensure the document contains no formatting other than Markdown. Take care when copy/pasting from other text editors.</p>
</blockquote>
</aside>
<p>Markdown may need a bit of explanation. It’s basically just text with a few extra characters for formatting, like enclosing words in asterisks (<code>*</code>) to make them <em>italic</em>. There’s a good introduction and reference here: <a href="https://guides.github.com/features/mastering-markdown/">https://guides.github.com/features/mastering-markdown/</a></p>

**Comment by kimgandy** [comment]
<p>YOU…ARE…AWESOME! Thank you so much!</p>
